lib/organisation_importer.rb

```diff
@@ -24,13 +24,16 @@ def create_or_update_organisation(organisation)
     if existing_tag.present?
       logger.info "Found existing tag: #{tag_id}"
 
-      if existing_tag.title != organisation.title
-        if existing_tag.update_attributes(title: organisation.title)
-          logger.info "Updated title to '#{existing_tag.title}'"
-        else
-          log_error_and_notify_airbrake(organisation,
-                                        "Could not update title: #{existing_tag.errors.full_messages}")
+      existing_tag.title = organisation.title
+      existing_tag.content_id = organisation.details.content_id
+
+      if existing_tag.save
+        if existing_tag.changes.any?
```

Comment:
That would not be the same (it would `log_error_and_notify_airbrake` if it's saved and there are no changes).

You're right about the code quality here, it's still a mess of nested conditions, but I don't want to refactor it just for this change, since panopticon will be going away at one point anyway.